pkg/reconciler/cronjobsource/resources/receive_adapter_test.go

```diff
@@ -111,14 +112,24 @@ func TestMakeReceiveAdapter(t *testing.T) {
 									Value: "source-namespace",
 								},
 							},
+							Resources: corev1.ResourceRequirements{
```

Comment:
It seems like we only test the case where the defaults are assigned. Could you add a test where the user specifies these limits and make sure they make it to the underlying deployment spec? Or, perhaps I missed something?

Reply:
Nope, great catch, thank you!
I've committed a test (following the same pattern we used in the kafka receive adapter) in be72f99